guides/common/modules/proc_migrating-project-or-smartproxy-using-clone.adoc

```diff
@@ -0,0 +1,24 @@
+[id="migrating-{project-context}-or-proxy-using-clone_{context}"]
+= Migrating {Project} or {SmartProxy} to {EL-abbr}{nbsp}9 by using cloning
+
+You can clone your existing {ProjectServer} from {EL}{nbsp}8 to a freshly installed {EL}{nbsp}9 system.
+ifdef::satellite[]
+Create a backup of the existing {ProjectServer}, which you then clone on the new {EL}{nbsp}9 system.
+[NOTE]
+====
+You can use cloning for {ProjectServer} backups but not for {SmartProxyServer} backups.
+====
+endif::[]
+
+.Procedure
+. Perform a full backup of your {ProjectServer}.
+This is the source {EL}{nbsp}8 server that you are migrating.
+For more information, see {AdministeringDocURL}backing-up-{project-context}-server-and-{smart-proxy-context}_admin#Performing_a_Full_Backup_admin[Performing a full backup of {ProjectServer}] in _{AdministeringDocTitle}_.
+. Deploy a system with {EL}{nbsp}9 and the same hostname and configuration as the source server.
+This is the target server.
+ifdef::satellite[]
+. Clone the server.
+Clone configures hostname for the target server.
```

Comment:
I don't get this sentence :see_no_evil: Perhaps "Cloning configures a hostname [...]"? But then again, aren't users supposed to define the hostname on line 17?

Reply:
Removed the hostname mention from line 17 as that's not necessary for cloning.